sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -2031,7 +2033,21 @@ object DatePart {
                  "MILLISECONDS", ("MSEC", "MSECS", "MILLISECON", "MSECONDS", "MS"),
                  "MICROSECONDS", ("USEC", "USECS", "USECONDS", "MICROSECON", "US"),
                  "EPOCH"]
-      * source - a date (or timestamp) column from where `field` should be extracted
+                Supported string values of `field` for intervals are:
```

Comment:
So, the difference is only excluding the following?
```
"ISOYEAR",
"WEEK", ("W", "WEEKS"),
"DAYOFWEEK",
"DOW",
"ISODOW",
"DOY",
```

Reply:
Yes, do you want to point out the difference only?

Reply:
For now, the enumeration also looks enough.